scripts/cnode-helper-scripts/env

```diff
@@ -411,6 +413,26 @@ checkUpdate() {
   return 0
 }
 
+# Description : Check that old script User Variables section contain all current settings from GitHub.
+#             : Prints final result to stdout
+#             : $1 = Old/Current script User Variables
+#             : $2 = User Variables section from GitHub
+checkUserVariables() {
+  tmp_head=$(head -n -2 <<< "$1")
+  tmp_tail=$(tail -n -2 <<< "$1")
+  while IFS= read -r line; do
+    if [[ ${line} =~ ^\#?([[:alnum:]_]+\=) ]]; then
```

Comment:
Should we add space char too? often users when commenting a variable could.do `# varname=..`
to avoid duplicates

Reply:
👍 improved regex to match both on multiple # and spaces before the variable